include/ibamr/RelaxationLSMethod.h

```diff
@@ -152,18 +157,29 @@ class RelaxationLSMethod : public IBAMR::LSInitStrategy
     /*!
      * \brief Do one relaxation step over the hierarchy.
      */
-    void relax(SAMRAI::tbox::Pointer<IBTK::HierarchyMathOps> hier_math_ops,
+    void relax(SAMRAI::tbox::Pointer<IBTK::HierarchyGhostCellInterpolation> D_fill_op,
+               SAMRAI::tbox::Pointer<IBTK::HierarchyMathOps> hier_math_ops,
                int dist_idx,
                int dist_init_idx,
-               const int iter) const;
+               int dist_copy_idx,
+               const int iter,
+               const double time) const;
+
+    /*!
+     * \brief Do Gauss-Seidel based one relaxation step over a patch.
```

Comment:
Can you add a brief comment here about how G-S is being used as a pseudo-time integrator?

Reply:
I added in here:

https://github.com/IBAMR/IBAMR/blob/amneetb/extending_LS_ex1/src/level_set/RelaxationLSMethod.cpp#L632-L650